.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,49 @@
+assignees: ''
+
+---
+
+### タイトル
+
+### 説明
+バグの詳細な説明を記入してください。
+
+### 再現手順
+バグを再現するための手順を記入してください。
+1. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)